blog post for cordova-android@7.1.4 patch release (#914)
apache/cordova-docs: www/_posts/2018-11-23-cordova-android-7.1.4.md

---
layout: post
author:
    name: Chris Brody
    url: https://twitter.com/brodybits
title:  "Cordova Android 7.1.4 Released!"
categories: announcements
tags: news releases
---

> We are happy to announce that `Cordova Android 7.1.4` has been released! This release resolves a couple more issues releated to plugin compatibility.

Reviewer: ```suggestion
We are happy to announce that `Cordova Android 7.1.4` has been released! This release resolves a couple more issues related to plugin compatibility.
```

> We are happy to announce that Cordova Android `7.1.4` has been released! This resolves an issue that was introduced in the recent `7.1.3` release. Thanks to the user community for the quick reports on GitHub issues and the help in identifying the problem.

Reviewer: ```suggestion
We are happy to announce that Cordova Android `7.1.4` has been released! This resolves a bug that was introduced in the recent `7.1.3` release. Thanks to our users for the quick reports on [GitHub issues](https://github.com/apache/cordova-android/issues) and the help in identifying the problem.
```
